pkg/agent/secondarynetwork/podwatch/controller.go

```diff
@@ -138,6 +138,9 @@ func NewPodController(
 		// Subscribe Pod CNI add/del events.
 		podUpdateSubscriber.Subscribe(pc.processCNIUpdate)
 	}
+
+	pc.RestoreSecondaryNetwork(ovsBridgeClient, ifaceStore)
```

Comment:
what about calling this function after the local cache is synced and before running the workers in goroutines https://github.com/antrea-io/antrea/blob/main/pkg/agent/secondarynetwork/podwatch/controller.go#L488  ? Then you can have check on the errors occurred in function `RestoreSecondaryNetwork` ?